src/main/java/hudson/plugins/svn_tag/SvnTagPublisher.java

```diff
@@ -43,11 +45,18 @@
 
     private String tagDeleteComment = null;
 
+    private Integer waitBeforeTagging = 0;

```

Comment:
You can use "int" and "0" value in your code. All checks seem to address such case